android/KMEA/app/src/main/assets/keyboard.html

```diff
@@ -235,6 +235,26 @@
       }
     }
 
+    function suggestionPopup(obj,custom,x,y,w,h) {
+      if(obj != null) {
+        var i;
+        var s = JSON.stringify(obj);
+        var shift = false;
+        var pos = x.toString() + ',' + y.toString();
+        var size = w.toString() + ',' + h.toString();
+        var frame = pos + ',' + size;
```

Comment:
`i`, `shift`, `pos`, `frame` and `size` now appear to be unused variables? Also we should be using `let` not `var` these days ;-)